master/usage/decommissioning-a-node.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Decommissioning a Calico Node
+---
+
+With each Calico node there is a [Node resource][Node resource reference]
```

Comment:
I think we really should be even more explicit (not just here, but across all of our docs).

A user should be able to spend <30s reading a doc to decide if they care about its contents.

```
You may be interested in this document if:
- You are decommissioning a node or removing it from your cluster.
- You are renaming a Node.
- You are encountering such and such an error.
```